<launch>

<node name="rp2w" pkg="ros_rp2w" type="js_rp2w" respawn="true" />

<node name="rospyopenni" pkg="ros_rp2w" type="rospyopenni.py" respawn="true" />

<node name="recnora" pkg="ros_rp2w" type="RecNora.py" respawn="true" />

<node name="VRec1" pkg="ros_rp2w" type="VRec1.0.py"  respawn="true" >

<param name="startTrainOrRec" value="rec" />
<param name="rateCounter" value="3" />


 </node>


<!--<node name="pygameface" pkg="ros_rp2w" type="pygameface.py" respawn="true" />-->

<node name="pyoutput" pkg="ros_rp2w" type="pyoutput.py" respawn="true" />

<node name="imageconverter" pkg="ros_rp2w" type="image_converter" respawn="true" />

<node name="My_Emotion" pkg="ros_rp2w" type="DLIBlandmarking.py" respawn="true" />

<node name="tts_service" pkg="ros_rp2w" type="festserv.py" respawn="true" />


<node name="speechrec" pkg="ros_rp2w" type="speechrec.py" output="log" respawn="true" />

<node name="Display" pkg="ros_rp2w" type="display.py"  respawn="true" />



</launch>
